sim/shaman/stormstrike.go

```diff
@@ -39,6 +40,23 @@ func (shaman *Shaman) newStormstrikeTemplate(sim *core.Simulation) core.MeleeAbi
 			MainHand: 1.0,
 			Offhand:  1.0,
 		},
+		OnMeleeAttack: func(sim *core.Simulation, target *core.Target, result core.MeleeHitType, ability *core.ActiveMeleeAbility, isOH bool) {
+			stacks := 2
+			target.ReplaceAura(sim, core.Aura{
+				ID:   StormstrikeDebuffID,
+				Name: "Stormstrike",
+				OnBeforeSpellHit: func(sim *core.Simulation, spellCast *core.SpellCast, spellEffect *core.SpellEffect) {
+					if spellCast.SpellSchool != stats.NatureSpellPower {
+						return
+					}
+					spellEffect.DamageMultiplier *= 1.2
+					stacks--
```

Comment:
This isn't going to work right for the raid sim, because SS from different shaman should just reset the stacks on that particular target.

Take a look at the new shadow weaving code for an example of doing this.